pylint/typing.py

```diff
@@ -57,7 +57,7 @@ class MessageLocationTuple(NamedTuple):
 
 
 class ManagedMessage(NamedTuple):
-    """Tuple with information ahout a managed message of the linter"""
+    """Tuple with information about a managed message of the linter"""
```

Comment:
Perhaps clean up the commit history and don't squash?

Reply:
You're right I wanted to rebase originally but there are other commits that prevents that now.